bitbots_dynup/src/dynup_node.cpp

```diff
@@ -137,7 +216,7 @@ void DynUpNode::publishGoals(const bitbots_splines::JointGoals &goals) {
   joint_goal_publisher_.publish(command);
 }
 void DynUpNode::publishSupportFoot(bool is_left_dyn_up) {
-  // Currently not implemented
+  //TODO: Currently not implemented
```

Comment:
Please open an issue for this todo.

Reply:
I don't think this is actually needed anymore, removed the todo.